tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/deleted.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- debug:
+    msg: Start iosxr_bgp_nbr_address_family deleted integration tests connection={{ ansible_connection}}
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate_base_config.yaml
+
+- include_tasks: _populate_config.yaml
+
```

Comment:
Please add a test case, that removes a single AF when base config has multiple ones. Both under top-level neighbors and neighbors under VRF.

Reply:
Ack,made changes